src/components/Home.jsx

```diff
@@ -23,7 +25,11 @@ export default function Home() {
   loading ? <p className='loading'>Loading</p> : (
     <main className='article-list'>
       {articles.map((article)=>{
```

Comment:
Filtering by topic should not be done on the front end, but by utilising back end queries. Have a look at the NC notes on axios for some pointers on this🙂